builtin/credential/ldap/backend.go

```diff
@@ -62,6 +65,8 @@ func Backend() *backend {
 
 type backend struct {
 	*framework.Backend
+
+	mu sync.RWMutex
```

Comment:
Might've missed it, but did we end up using this?

Reply:
we don't currently - but I think we will - i'm still straightening out when we need to lock

Reply:
this is used now for protecting writes of the config